gunfolds/conversions.py

```diff
@@ -501,10 +501,16 @@ def numbered_g2wclingo(g, num, directed_weights_matrix=None, bidirected_weights_
     """
     s = ''
     n = len(g)
+
     if directed_weights_matrix is None:
         directed_weights_matrix = np.ones((n, n)).astype('int')
+    else:
```

Comment:
would it not be shorter, easier, to get rid of this line `else` and just leave the next line making it execute in either case. Also in this case would change `directed_weights_matrix = np.ones((n, n)).astype('int')` to `directed_weights_matrix = np.ones((n, n))` just shorter